marketplace_builder/views/pages/tutorials/data-import-export/import.liquid

```diff
@@ -69,16 +67,14 @@ marketplace-kit data import --path=data.json staging-instance
 
 It is possible to import data using GraphQL mutations. This gives you more flexibility, and even allows you to create your own import tool as a website feature. Please remember to handle authentication properly.
 
-There are three main mutations for importing [import_users](), [import_models]() and [import_transactables](). You can create a page that will import sent data.
+There are three main mutations for importing [import_users](), [import_models](). You can create a page that will import sent data.
```

Comment:
```suggestion
There are three main mutations for importing import_users, import_models. You can create a page that will import sent data.
```

Reply:
Now that we've removed Transactables, there are only two, if I understand it correctly. So I'd write: 

There are two main mutations for importing: `import_users` and `import_models`.

Reply:
I updated suggestion.

My original one was about having links that have empty href, which is not helpful on the world wide web.